src/main/java/de/tum/in/www1/artemis/web/rest/ModelingAssessmentResource.java

```diff
@@ -231,6 +271,12 @@ public Result updateManualResult(Long resultId, Long exerciseId, String modeling
         return result;
     }
 
+    /**
+     * Helper function to calculate the total score of an assessment.
+     *
+     * @param assessmentJson    the assessments as JsonObject
+     * @return the total score
+     */
     public Double calculateTotalScore(JsonObject assessmentJson) {
```

Comment:
This helper function should be commented better and should rather be moved to an AssessmentService